utils/spdx/src/main/kotlin/SpdxExpression.kt

```diff
@@ -411,7 +411,7 @@ class SpdxLicenseWithExceptionExpression(
     val exception: String
 ) : SpdxSingleLicenseExpression() {
     companion object {
-        private val EXCEPTION_STRING_PATTERN = Regex("\\bexception\\b", RegexOption.IGNORE_CASE)
+        private val EXCEPTION_STRING_PATTERN = Regex("\\b(exception|additional-terms)\\b", RegexOption.IGNORE_CASE)
```

Comment:
I'd prefer to stay on the safe-side and list `LicenseRef-scancode-agpl-generic-additional-terms` explicitly, to not risk matching anything else. What do you think?

Reply:
That would miss at least `LicenseRef-scancode-gpl-generic-additional-terms` (`gpl` vs `agpl`) which exists, too. In any case, I believe it's safe enough to assume something to be an exception that if it

1. is used with the `WITH` operator, and
2. starts with "LicenseRef-", and
3. contains "additional-terms"

Reply:
Out of scope:

Looking into `ScanCode`'s data it turns out that this identifier has `is_generic = true` which implies that there is not license text defined for that license identifier. (So, it's similar to this LicenseRef-scancode-other-permissive and so forth). Just wanted to share this, I don't have any idea on how this could be best addressed. 

My guess would be that we need a mechanism which omits generic licenses from SPDX reports, for both license IDs and exception IDs, as it does not make sense to list license IDs which are undefined in terms of the corresponding text.